src/http/invoke-http/_res-fmt.js

```diff
@@ -9,20 +9,31 @@ module.exports = function responseFormatter ({res, result}) {
   // Content type
   // Note: result.headers is a case-sensitive js object,
   //       res.get/setHeader is not (e.g. 'set-cookie' == 'Set-Cookie')
-  let contentType = result && result.headers && result.headers['Content-Type'] ||
+  let contentType = result && result.multiValueHeaders && result.multiValueHeaders['Content-Type'] ||
+                    result && result.multiValueHeaders && result.multiValueHeaders['content-type'] ||
+                    result && result.headers && result.headers['Content-Type'] ||
                     result && result.headers && result.headers['content-type']
   res.setHeader('Content-Type', contentType || 'application/json; charset=utf-8')
-  if (result.headers && result.headers['content-type']) delete result.headers['content-type']
+  if (result.multiValueHeaders && result.multiValueHeaders['content-type'])
+    delete result.multiValueHeaders['content-type']
+  else if (result.headers && result.headers['content-type']) {
+    delete result.headers['content-type']
+  }
 
   // Headers
-  if (result.headers) {
-    Object.keys(result.headers).forEach(k=> {
-      if (k.toLowerCase() === 'set-cookie' && result.headers[k]) {
-        res.setHeader(k, result.headers[k].replace('; Secure', '; Path=/'))
-      } else if (k === 'cache-control' && result.headers[k]) {
-        res.setHeader('Cache-Control', result.headers[k])
+  const headers = result.multiValueHeaders || result.headers
```

Comment:
What's the expected behavior in AWS land if you supply BOTH `headers` and `multiValueHeaders`? Will it pass back both, combined, or does it follow the above logic privileging `multiValueHeaders` over `headers?

Reply:
Great watch! I wasn't sure and was even more surprised by the actual behaviour!

This code:
```
exports.handler = async function http() {
  return {
    headers: {
      'Test1': 'Foo',
      'Test2': 'Foobar',
    },
    multiValueHeaders: {
      'Test1': ['Bar', 'Baz'],
      'Test3': ['Foobar'],
    },
    body: 'Hello',
  };
}
```

Returns this response:
```
HTTP/1.1 200 OK
Connection: keep-alive
Content-Encoding: gzip
Content-Length: 25
Content-Type: application/json
Date: Sun, 19 Apr 2020 12:26:03 GMT
Test1: Bar
Test1: Baz
Test1: Foo
Test2: Foobar
Test3: Foobar
X-Amzn-Trace-Id: Root=1-5e9c435b-68cc1b34e7f4ecbc7640f39e;Sampled=0
x-amz-apigw-id: LO92QGzcvHcFuEw=
x-amzn-RequestId: 0c8062f4-669e-4d68-935e-f0071aeb5cc8

Hello
```

I'll have a go at modifying Sandbox to adopt this behaviour.